### ISSUE TITLE
Issue Installing Packages (ST2)

### ISSUE THREAD

**Issue by cashewguy** [wbond/package_control#931]
Details:
OS X Yosemite 10.10.3
Sublime Text 2 Version 2.0.2, Build 2221
Package Control V3.0.0.

I can't seem to install or update any packages. I'm trying to install SublimeGit.

Here is the console output when I try something:

```
Traceback (most recent call last):
  File "/Users/thomas/Library/Application Support/Sublime Text 2/Packages/Package Control/package_control/package_installer.py", line 169, in on_done
    if name in self.disable_packages(name, 'install'):
  File "/Users/thomas/Library/Application Support/Sublime Text 2/Packages/Package Control/package_control/package_disabler.py", line 103, in disable_packages
    if package_file_exists(package, settings.get('theme')):
  File "/Users/thomas/Library/Application Support/Sublime Text 2/Packages/Package Control/package_control/package_io.py", line 33, in package_file_exists
    file_path = os.path.join(package_dir, relative_path)
  File "/System/Library/Frameworks/Python.framework/Versions/2.6/lib/python2.6/posixpath.py", line 65, in join
    if b.startswith('/'):
AttributeError: 'NoneType' object has no attribute 'startswith'
```

I was able to install the package by adding it to the list of installed packages then restarting - however, I still can't upgrade packages or install other ones.

**Comment by cashewguy** [comment]
Problem persists with any package on package control. Here's the output for installing 2pdf:

```
Traceback (most recent call last):
  File "/Users/thomas/Library/Application Support/Sublime Text 2/Packages/Package Control/package_control/package_installer.py", line 169, in on_done
    if name in self.disable_packages(name, 'install'):
  File "/Users/thomas/Library/Application Support/Sublime Text 2/Packages/Package Control/package_control/package_disabler.py", line 103, in disable_packages
    if package_file_exists(package, settings.get('theme')):
  File "/Users/thomas/Library/Application Support/Sublime Text 2/Packages/Package Control/package_control/package_io.py", line 33, in package_file_exists
    file_path = os.path.join(package_dir, relative_path)
  File "/System/Library/Frameworks/Python.framework/Versions/2.6/lib/python2.6/posixpath.py", line 65, in join
    if b.startswith('/'):
AttributeError: 'NoneType' object has no attribute 'startswith'
```

(that's with debug mode on).

---

Since the only related issue (or most closely similar issue) I can find is [this](https://github.com/wbond/package_control/issues/675): This is my Settings - User file:

```
{
	"debut": true,
	"installed_dependencies":
	[
		"0_package_control_loader",
		"bz2"
	],
	"installed_packages":
	[
		"BracketHighlighter",
		"ColorPicker",
		"Emmet",
		"Handlebars",
		"HTML Boilerplate",
		"HTML2Text",
		"HTML5",
		"HTML5 Doctor CSS Reset snippet",
		"Jade",
		"Package Control",
		"SCSS",
		"SFTP",
		"Solarized Color Scheme",
		"Solarized Toggle",
		"SublimeGit",
		"Theme - Soda SolarizedDark",
		"Theme - Sodarized",
		"Whitespace",
		"Whitespaces"
	]
}
```

**Comment by wbond** [comment]
Sorry that I missed this bug report. It appears the issue is that your `theme` setting is currently blank. If you set that to `Default.sublime-theme`, it should work again.

**Comment by wbond** [comment]
This should also be fixed by b42dfb492112176e1b166eda74694e6fd935e534
